package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-fadethrough",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
Don't change this, I'll do a standalone commit to update the version number and publish to npm.

Reply:
Got it.